src/main/java/org/opensearch/ad/transport/EntityResultTransportAction.java

```diff
@@ -158,14 +162,14 @@ private ActionListener<Optional<AnomalyDetector>> onGetDetector(
     ) {
         return ActionListener.wrap(detectorOptional -> {
             if (!detectorOptional.isPresent()) {
-                listener.onFailure(new EndRunException(detectorId, "AnomalyDetector is not available.", true));
+                listener.onFailure(new EndRunException(detectorId, "AnomalyDetector is not available.", false));
```

Comment:
Why change to `false`? If detector is not found, do we still need to run job?

Reply:
It might be the node with the detector config is temporarily unavailable. Changing to false to to accommodate that.